projects/dsp-ui/src/lib/viewer/services/value-type.service.spec.ts

```diff
@@ -46,4 +46,43 @@ describe('ValueTypeService', () => {
             expect(service.isReadOnly(valueClass)).toBeTruthy();
         });
     });
+
+    describe('compareObjectTypeWithValueType', () => {
+
+        it('should successfully compare "http://api.knora.org/ontology/knora-api/v2#TextValue" with "ReadTextValueAsString"', () => {
+            expect(service.compareObjectTypeWithValueType(
+                'ReadTextValueAsString',
+                'http://api.knora.org/ontology/knora-api/v2#TextValue'))
+                .toBeTruthy();
```

Comment:
Could you maybe add another (failed) comparison between a text value and a non text type?

Reply:
text value meaning `http://api.knora.org/ontology/knora-api/v2#TextValue` or `ReadTextValueAsString`?

Reply:
compare a `ReadTextValueAsString` to an IntValue for example. Sorry for being unprecise

Reply:
no worries, added in 691a5a8 :)

Reply:
thx, good to go then!